Mlem/Enums/FeedType.swift

```diff
@@ -5,16 +5,34 @@
 //  Created by Jonathan de Jong on 12.06.2023.
 //
 
-import Foundation
+import SwiftUI
 
 enum FeedType: String, Encodable, SettingsOptions {
     var id: Self { self }
 
     var label: String {
+        return rawValue
+    }
+    
+    var description: String {
+        switch self {
+        case .all:
+            return "Subscribed communities from all instances"
+        case .local:
+            return "Local communities from your server"
+        case .subscribed:
+            return "All communities that federate with your server"
+        }
+    }
+    
+    var color: Color {
```

Comment:
`AssociatedColor` conformance

```suggestion
    var color: Color? {
```